TravelerAPI/TravelerAPI/Currency/View/SymbolsView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ListCurrenciesViewController.swift
+//  TravelerAPI
+//
+//  Created by Marc-Antoine BAR on 2022-10-28.
+//
+
+import UIKit
+
+final class SymbolsViewController: UIViewController {
+
+    // MARK: Properties
+    private let mytableView = UITableView()
+    private let identifierCell = "myCell"
+    private var Lisymbols = [String]()
+    
+    // - MARK: Cycle Method
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = "Choose Currency"
+        setupStyle()
+        setupSubView()
+        setupLayout()
+        
+        // FIXME: msut go in the models !
```

Comment:
`#warning("FIXME: must go in the models !")`
là tu es sur de ne pas l'oublier ;)